s3/replication/common/src/s3replicationcommon/s3_session.py

```diff
@@ -31,7 +32,8 @@ def __init__(self, logger, s3_site, access_key, secret_key):
         self.access_key = access_key
         self.secret_key = secret_key
 
-        self._client_session = aiohttp.ClientSession()
+        connector = aiohttp.TCPConnector(limit=number_of_connections)
```

Comment:
Do we require similar changes at other 'aiohttp.ClientSession' instances as well?

Reply:
This is currently for all S3 connections, for other paths (manager <--> replicator) we will add in future and will need to be tested separately.